EpikCord/application/application.py

```diff
@@ -1,26 +1,27 @@
-from typing import Optional
+from typing import List, Optional
+
 from ..partials import PartialUser
 from .team import Team
 
 
 class Application:
     def __init__(self, data: dict):
-        self.id: str = data.get("id")
-        self.name: str = data.get("name")
+        self.id: int = int(data["id"])
+        self.name: str = data["name"]
         self.icon: Optional[str] = data.get("icon")
-        self.description: str = data.get("description")
-        self.rpc_origins: Optional[list] = data.get("rpc_origins")
-        self.bot_public: bool = data.get("bot_public")
-        self.bot_require_code_grant: bool = data.get("bot_require_code_grant")
+        self.description: str = data["description"]
+        self.rpc_origins: Optional[List[str]] = data.get("rpc_origins")
+        self.bot_public: bool = data["bot_public"]
+        self.bot_require_code_grant: bool = data["bot_require_code_grant"]
         self.terms_of_service_url: Optional[str] = data.get("terms_of_service")
         self.privacy_policy_url: Optional[str] = data.get("privacy_policy")
         self.owner: Optional[PartialUser] = (
-            PartialUser(data.get("user")) if data.get("user") else None
+            PartialUser(data["user"]) if "user" in data else None
         )
-        self.verify_key: str = data.get("verify_key")
-        self.team: Optional[Team] = Team(data.get("team")) if data.get("get") else None
+        self.verify_key: str = data["verify_key"]
+        self.team: Optional[Team] = Team(data["team"]) if data["team"] else None
```

Comment:
Wouldn't this return a KeyError?

Reply:
![image](https://user-images.githubusercontent.com/34910574/184629403-734c29ee-0701-4eef-91cf-241f02b1ac8f.png)

The `?` is on the type which means it is nullable (`T | None`).
This creates a `Team` object if the `team` field is not `None`

Reply:
Yes but the `if date["team"] else None` would raise a KeyError
```py
>>> data = {} # The key doesn't exist
>>> bar = int(data["foo"]) if data["foo"] else None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
KeyError: 'foo'```

Reply:
What I'm showing you is `?type` means the key **will exist**, but will just be **nullable**.

Reply:
My bad, I thought that was about the key.